modules/metrics/src/test/scala/surge/metrics/TimerSpec.scala

```diff
@@ -22,7 +22,7 @@ class TimerSpec extends MetricsSpecLike with ScalaFutures {
       metricValue(testTimerName) shouldEqual 95.5
     }
 
-    "Properly time scala Future completion time" in {
+    "Properly time scala Future completion time" ignore {
```

Comment:
I think these can be un-ignored, they should be fixed (or more stable) with the latest commit on main.

Reply:
@jeffboutotte Awesome, I'll unignore.